CrowdFundingApplication/Services/EmailService.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace CrowdFundingApplication.Services;
+public class EmailService
+{
+    public async Task SendEmailAsync(string email, string subject, string message)
+    {
+        var emailMessage = new MimeMessage();
+
+        emailMessage.From.Add(new MailboxAddress("Администрация платформы ViewStar", "viewstarsettings@gmail.com"));
+        emailMessage.To.Add(new MailboxAddress("", email));
+        emailMessage.Subject = subject;
+        emailMessage.Body = new TextPart(MimeKit.Text.TextFormat.Html)
+        {
+            Text = message
+        };
+
+        using (var client = new MailKit.Net.Smtp.SmtpClient())
+        {
+            await client.ConnectAsync("smtp.gmail.com", 587, false);
+            await client.AuthenticateAsync("viewstarsettings@gmail.com", "899adminviewstar");
```

Comment:
You can’t do this, the secret data must be put in the config and not committed to GIT, but set through environment variables or read from the secret storage.

Reply:
Changed